lib/pangea/activity_sessions/activity_session_start/activity_sessions_start_view.dart

```diff
@@ -153,8 +154,11 @@ class ActivitySessionStartView extends StatelessWidget {
                                 ),
                                 onPressed: () => showFutureLoadingDialog(
                                   context: context,
-                                  future: () => controller.room
-                                      .invite(BotName.byEnvironment),
+                                  future: () async {
+                                    controller.room
+                                        .invite(BotName.byEnvironment);
+                                    controller.isBotRoomMember = true;
```

Comment:
Setting the value of a state variable from the view this way is brittle and tends to cause issues. For instance, what if someone else in the chat invited the bot? ```controller.isBotRoomMember``` wouldn't be set to true in that case.

In this case, you can just check if the bot is in the room synchronously, since the full list of members is loaded in chat.dart and changes to the members are updated by the stream builder. It would look like:

```
  bool get isBotRoomMember => room.getParticipants().any(
        (p) => p.id == BotName.byEnvironment,
      );
```

And then it won't have to be stored in a state variable